src/toil/provisioners/aws/awsProvisioner.py

```diff
@@ -341,7 +341,7 @@ def descriptionMatches(ami):
         for attempt in retry(predicate= lambda e : isinstance(e, SSLError)):
             # SSLError is thrown when get_all_images times out
             with attempt:
-                amis = self._ctx.ec2.get_all_images(owners=['679593333241'])
+                amis = self._ctx.ec2.get_all_images(image_ids=['ami-70d5b608'])
```

Comment:
Really good point, I'll test it!  Thanks!

Reply:
You were right, it was only passing because it was in the correct region.  I've reverted it to the old code, which is the reference to the aws marketplace.  The latest version I tested this with I thought had errored on that "user" but now I suspect it may have been something different.

Anyway, should be fixed.  Thanks for catching my error!

Reply:
No worries--I only know this because I made this mistake before!